src/main/webapp/app/components/tabs/CurationHistoryTab.tsx

```diff
@@ -123,10 +123,10 @@ const CurationHistoryTab = observer(({ historyData, usersData, addUsersListener,
 
       content.push(
         <div>
-          <Row className="justify-content-between" style={{ marginBottom: '22px' }}>
+          <div className="d-flex justify-content-between align-items-center" style={{ marginBottom: '22px' }}>
             <span>
               <TextFormat value={data.createdAt} type="date" format={APP_HISTORY_FORMAT} />
-              <span className="mr-2" />
+              <span className="me-2" />
```

Comment:
Looks like that starting using start and end rather than left and right to support RTL layouts?

Reply:
Yeah, exactly